src/pages/new-lesson/NewLesson.tsx

```diff
@@ -1,5 +1,142 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import useForm from '~/hooks/use-form'
+import useAxios from '~/hooks/use-axios'
+import { Link, useNavigate } from 'react-router-dom'
+
+import { Editor } from '@tinymce/tinymce-react'
```

Comment:
separate imports from libraries and local imports of the components

Reply:
fixed